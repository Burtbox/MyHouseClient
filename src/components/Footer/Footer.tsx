import * as React from 'react';
import styles from './footerStyles';

const Footer: React.StatelessComponent = () => {
    return (
        <footer style={styles.footer}>
            <div>
                By hApps, Version 1.0
                </div>
            <div> Icons made by
                    <a href="https://www.flaticon.com/authors/dave-gandy"
                    title="Dave Gandy"> Dave Gandy
                    </a> from
                    <a href="https://www.flaticon.com/"
                    title="Flaticon"> www.flaticon.com
                    </a> is licensed by
                    <a href="http://creativecommons.org/licenses/by/3.0/"
                    title="Creative Commons BY 3.0"
                    target="_blank"> CC 3.0 BY
                    </a>
            </div>
        </footer>
    );
};

export default Footer;
