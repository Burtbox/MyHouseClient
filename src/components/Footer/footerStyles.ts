const styles: { [name: string]: React.CSSProperties } = {
    footer: {
        textAlign: 'center',
        padding: '20px',
        position: 'fixed',
        left: '0',
        bottom: '0',
        height: '60px',
        width: '100%',
    },
};

export default styles;
