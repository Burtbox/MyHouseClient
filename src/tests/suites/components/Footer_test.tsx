import * as React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../../components/Footer';

describe('Footer test suite', () => {
    test('Render Footer component', () => {
        const FooterScreen = shallow(
            <Footer />,
        );
        expect(FooterScreen).toMatchSnapshot();
    });
});
