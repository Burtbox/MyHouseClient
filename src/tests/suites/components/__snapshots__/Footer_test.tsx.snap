// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer test suite Render Footer component 1`] = `
<footer
  style={
    Object {
      "bottom": "0",
      "height": "60px",
      "left": "0",
      "padding": "20px",
      "position": "fixed",
      "textAlign": "center",
      "width": "100%",
    }
  }
>
  <div>
    By hApps, Version 1.0
  </div>
  <div>
     Icons made by
    <a
      href="https://www.flaticon.com/authors/dave-gandy"
      title="Dave Gandy"
    >
       Dave Gandy
    </a>
     from
    <a
      href="https://www.flaticon.com/"
      title="Flaticon"
    >
       www.flaticon.com
    </a>
     is licensed by
    <a
      href="http://creativecommons.org/licenses/by/3.0/"
      target="_blank"
      title="Creative Commons BY 3.0"
    >
       CC 3.0 BY
    </a>
  </div>
</footer>
`;
